### ISSUE TITLE
priors should have unique names after set_confound

### ISSUE THREAD

**Issue by macartan** [macartan/gbiqq#47]
see repetition of X.X0 and X.X1 below

```
> model <- make_model("X -> Y <- S; S -> W") %>%
+          set_confound(list(X = "W==1"))
> 
> 
> model$priors
   X.X0    X.X1    S.S0    S.S1    X.X0    X.X1   W.W00   W.W10   W.W01   W.W11 Y.Y0000 Y.Y1000 Y.Y0100 Y.Y1100 Y.Y0010 Y.Y1010 Y.Y0110 Y.Y1110 
      1       1       1       1       1       1       1       1       1       1       1       1       1       1       1       1       1       1 
Y.Y0001 Y.Y1001 Y.Y0101 Y.Y1101 Y.Y0011 Y.Y1011 Y.Y0111 Y.Y1111 
      1       1       1       1       1       1       1       1 
```

**Comment by lilymedina** [comment]
This issue is being addressed in PR [#48](https://github.com/macartan/gbiqq/pull/48)

**Comment by lilymedina** [comment]
solved in PR [#48](https://github.com/macartan/gbiqq/pull/48)
